Riva0110/Vocab-Garden
# Feature/article

### Description

* Add article-related pages
* User can add, edit and read articles
* User can search vocab by clicking any vocab from articles

##### Test Plan

* Randomly click vocab from articles
* Check if audio works by clicking audio images
* Check article-related routers are correct
